.circleci/config.yml

```diff
@@ -66,10 +66,6 @@ jobs:
           name: Run UI Test
           command: |
             ./gradlew app:connectedDebugAndroidTest
-      - run:
-          name: Run Unit Test
-          command: |
-            ./gradlew app:testDebugUnitTest
```

Comment:
Why is this disabled?

Reply:
Because was Unit Testing in Demo App, but there is no cases of this, only add more time in CI.
The reason of this is off Demo App only shows views, there is no logic. The E2E testing lives here.